src/deps/PDFWriter/DocumentContext.cpp

```diff
@@ -2176,7 +2176,7 @@ EStatusCode DocumentContext::SetupModifiedFile(PDFParser* inModifiedFileParser)
     if(idArray.GetPtr() && idArray->GetLength() == 2)
     {
         PDFObjectCastPtr<PDFHexString> firstID = idArray->QueryObject(0);
-        if(firstID.GetPtr())
+        if(firstID != NULL && firstID.GetPtr())
```

Comment:
verifying: is the null check really needed here? won't GetPtr() be falsy in this case?